lib/pymedphys/tests/dicom/test_base.py

```diff
@@ -28,7 +28,7 @@ def test_copy():
     dont_change_string = "don't change me"
     to_be_changed_string = "do change me"
 
-    new_manufactuer = "george"
+    new_manufacturer = "george"
```

Comment:
A fly-by typo fix.

Reply:
yes, but who is george?

Reply:
https://en.wikipedia.org/wiki/George_MacDonald :slightly_smiling_face: only my favourite author :slightly_smiling_face: